### ISSUE TITLE
WASD movement doesn't work

### ISSUE THREAD

**Issue by brndd** [tgstation/tgstation#39719]
Hello, I tried playing on /tg/station recently and the WASD movement wouldn't work. Only the A button would move my character. I tried fixing it, but it only caused my character to speak gibberish. Can you fix this?

**Comment by brndd** [comment]
thank you

**Comment by brndd** [comment]
no before I press tab the characters just go into the bar at the bottom

**Comment by ExcessiveUseOfCobblestone** [comment]
you have to click the ingame screen sometimes when that occurs

**Comment by brndd** [comment]
I don't think it's because of clicking on the screen. The problem is that only the A button moves my charactel. When I press S it opens the OOC chat, and when I press D it tries to equip items. W doesn't seem to do anything. Please fix this. It's unplayable.

**Comment by ExcessiveUseOfCobblestone** [comment]
This needs to be reproduced, I've never had this happen before and not sure what could cause it.

**Comment by deathride58** [comment]
Make sure you're using a QWERTY keyboard layout. BYOND does not support dvorak for input, which is what it sounds like you're trying to use.

**Comment by brndd** [comment]
Hello. I've played on other servers where the WASD movement worked. I had to edit the file (it was easy) but then it worked fine. On this server at first I could not find the file, but then I edited the execution file, and then it was there. However, changing the file did not work. I think it is this bug: #38006.

As said, changing the movement in Windows is not good, because then my character will only speak gibberish.
